src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/domene/IverksettData.kt

```diff
@@ -38,12 +38,15 @@ sealed class IverksettData {
     abstract val søker: Søker
     abstract val vedtak: Vedtaksdetaljer
 
-    fun erGOmregning(): Boolean = behandling.behandlingÅrsak == BehandlingÅrsak.G_OMREGNING
+    fun erGOmregning() = behandling.behandlingÅrsak == BehandlingÅrsak.G_OMREGNING
```

Comment:
Kan være fint å sjekke hvor denne blir brukt, er mulig vi trenger en `erGOmregningEllerSatsendring` ?

Reply:
Må beholde `erGOmregning()` fordi det er et eget flagg i oppdrag/utbetalingsmelding

Reply:
Skal satsendring også sende dette flagg til oppdrag?

Reply:
erGOmregningEllerSatsendring:
Ser ikke ut til at det er nødvendig ut ifra dette svaret: https://nav-it.slack.com/archives/C01EH16LAD9/p1668070501098049?thread_ts=1668069024.576789&cid=C01EH16LAD9

førstePubliseringsflytTask: Enig, satsendring er nok ikke nødvendig å publisere til så mye annet enn vedtakstatistikk